lib/playground.dart

```diff
@@ -691,10 +691,16 @@ class Playground implements GistContainer, GistController {
     String htmlSrc = _context.htmlSource.trim();
     String dartSrc = _context.dartSource.trim();
 
+    final List<String> webImports = [
+      "import 'dart:html'",
+      'import "dart:html"',
+      "import 'package:flutter_web",
+      'import "package:flutter_web',
+    ];
+
     if (htmlSrc.isNotEmpty) {
       outputTabController.selectTab('result');
-    } else if (dartSrc.contains("'dart:html'") ||
-        dartSrc.contains('"dart:html"')) {
+    } else if (webImports.any((String import) => dartSrc.contains(import))) {
```

Comment:
FWIW, I'd been hoping to leave flutter_web out of playground entirely until after it's incorporated into the new embed UI and tested there. Is that different than what you're working toward?

Reply:
I've commented out the flutter_web code. I'm not entirely sure how we'll limit support to certain front-ends; perhaps just add simple checks in the front-ends themselves.

Reply:
That's what I was thinking -- letting the front-end be in charge is probably fine for now.